### DISCUSSION TITLE
pcduino 3 no local login or no network

### DISCUSSION THREAD

**Discussion by vanneop** [johang/sd-card-images#15] [Q&A]
I am trying to install Debian Bullseye on my pcduino 3. With the pre-2022-09-26 images I had access to the local login on my hdmi monitor. However, I was not able to activate the eth0 connection. After creating a new image with the 2022-10-01 boot image and 2022-09-26 bullseye image, I am able to login through ssh, since the eth0 connection is active. However, there is nothing on the screen of my monitor. Since my goal is to have a locally accessible pcduino, with my keyboard and mouse connected through a usb hub and my monitor through hdmi, I tried to make the network connection to work, by downloading the network-manager files with all dependencies on pcduino with the "new" image, transferring them to a usb drive, which I then used to install on pcduino with the old image. This unfortunately does not work. I suspect that something relatively simple is causing this, but my knowledge is rather limited. 

Any help/pointer/suggestion would be appreciated.

**Comment by deftdawg** [comment]
These are the steps I did to get a PCDuino3 up and running a couple of days ago:
```sh
# Installing to SD Card on /dev/sda
wget http://dl.sd-card-images.johang.se/boots/2022-12-01/boot-pcduino3_nano.bin.gz
wget http://dl.sd-card-images.johang.se/debians/2022-12-05/debian-bookworm-armhf-eesh7i.bin.gz
zcat boot-pcduino3_nano.bin.gz debian-bookworm-armhf-eesh7i.bin.gz > sd-card.img
sudo dd if=sd-card.img of=/dev/sda bs=1M

# Mount the SD Card's root partition and add the system's SSH authorized_keys to the root user
mkdir /tmp/target
sudo mount /dev/sda2 /tmp/target
cd /tmp/target
# sudo cp ~/.ssh/authorized_keys root/.ssh/authorized_keys # copy all your Auth keys to the target install
cat ~/.ssh/id_rsa.pub | sudo tee -a root/.ssh/authorized_keys # copy your RSA pubkey to the target install
umount /tmp/target

# Boot the PCDuino3 connected to ethernet with a router that hands out DHCP Addresses
# Check the router for the DHCP Address of the PCDuino3 (default hostname is `debian`)
ssh root@debian # or @ the IP address from your router
```

**Comment by vanneop** [comment]
Thanks very much for your help.

**Reply by deftdawg** [reply]
My PCduino 3 seems to kernel crash about 21 hrs after it starts...  Would be curious to know if you see similar instability...  I still need to try some other stuff like attaching a serial and swapping for a different SD card.

**Comment by vanneop** [comment]
Unfortunately I use my PCduino 3 as a print server and turn it on and off
just to print. It usually takes only a few minutes, so I am not able to
confirm the issue you are having.

**Reply by deftdawg** [reply]
No worries 🍻
